pkg/gits/commits_test.go

```diff
@@ -36,91 +39,95 @@ func assertParseCommit(t *testing.T, input string, expected *CommitInfo) {
 }
 
 func TestChangelogMarkdown(t *testing.T) {
-	releaseSpec := &v1.ReleaseSpec{
-		Commits: []v1.CommitSummary{
-			{
-				Message: "some commit 1\nfixes #123",
-				SHA:     "123",
-				Author: &v1.UserDetails{
-					Name:  "James Strachan",
-					Login: "jstrachan",
+	if os.Getenv("RUN_UNENCAPSULATED_TESTS") == "true" {
```

Comment:
I would wrap this if into a closure just to make things more readable:

```Go
type UnencapsulatedFunc func()

func RunUnencapsulated(t *testing.T, func UnencapsulatedFunc) {
if os.Getenv("RUN_UNENCAPSULATED_TESTS") == "true" {
    func()
} else {
  t.Skip("skipping TestChangelogMarkdown; RUN_UNENCAPSULATED_TESTS not set")
}
```

and then in tests

```Go
RunUnencapsulated(t, func() {
// test code
})
```